<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>publications | Jazmín Ramos Madrigal</title> <meta name="author" content="Jazmín Ramos Madrigal"> <meta name="description" content="publications in reversed chronological order."> <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700%7CRoboto+Slab:100,300,400,500,700%7CMaterial+Icons"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%E2%92%BF&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css"> <link rel="canonical" href="https://jazminrm.github.io/publications/"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js"></script> <script src="/assets/js/dark_mode.js"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"><span class="font-weight-bold">Jazmín </span>Ramos Madrigal</a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about</a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications<span class="sr-only">(current)</span></a> </li> <li class="nav-item "> <a class="nav-link" href="/projects/">projects</a> </li> <li class="nav-item "> <a class="nav-link" href="/outreach/">outreach</a> </li> <li class="nav-item "> <a class="nav-link" href="/teaching/">teaching</a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">cv</a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description">publications in reversed chronological order.</p> </header> <article> <div class="publications"> <h2 class="year">2022</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Garcia-Ortiz2022-dq" class="col-sm-8"> <div class="title">Unraveling Signatures of Local Adaptation among Indigenous Groups from Mexico</div> <div class="author"> Humberto Garcı́a-Ortiz, Francisco Barajas-Olmos, Cecilia Contreras-Cubas, and <span class="more-authors" title="click to view 24 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '24 more authors' ? 'Austin W Reynolds, Marlen Flores-Huacuja, Meradeth Snow, Jazmı́n Ramos-Madrigal, Elvia Mendoza-Caamal, Paulina Baca, Tomás A López-Escobar, Deborah A Bolnick, Silvia Esperanza Flores-Martı́nez, Rocio Ortiz-Lopez, Aleksandar David Kostic, José Rafael Villafan-Bernal, Carlos Galaviz-Hernández, Federico Centeno-Cruz, Alejandra Guadalupe Garcı́a-Zapién, Tulia Monge-Cázares, Blanca Patricia Lazalde-Ramos, Francisco Loeza-Becerra, Marı́a Del Carmen Abrahantes-Pérez, Héctor Rangel-Villalobos, Martha Sosa-Macı́as, Augusto Rojas-Martı́nez, Angélica Martı́nez-Hernández, Lorena Orozco' : '24 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">24 more authors</span> </div> <div class="periodical"> <em>Genes</em>, Nov 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Few studies have addressed how selective pressures have shaped the genetic structure of the current Native American populations, and they have mostly limited their inferences to admixed Latin American populations. Here, we searched for local adaptation signals, based on integrated haplotype scores and population branch statistics, in 325 Mexican Indigenous individuals with at least 99% Native American ancestry from five previously defined geographical regions. Although each region exhibited its own local adaptation profile, only PPARG and AJAP1, both negative regulators of the Wnt/βcatenin signaling pathway, showed significant adaptation signals in all the tested regions. Several signals were found, mainly in the genes related to the metabolic processes and immune response. A pathway enrichment analysis revealed the overrepresentation of selected genes related to several biological phenotypes/conditions, such as the immune response and metabolic pathways, in agreement with previous studies, suggesting that immunological and metabolic pressures are major drivers of human adaptation. Genes related to the gut microbiome measurements were overrepresented in all the regions, highlighting the importance of studying how humans have coevolved with the microbial communities that colonize them. Our results provide a further explanation of the human evolutionary history in response to environmental pressures in this region.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Bergstrom2022-aq" class="col-sm-8"> <div class="title">Grey wolf genomic history reveals a dual ancestry of dogs</div> <div class="author"> Anders Bergström, David W G Stanton, Ulrike H Taron, and <span class="more-authors" title="click to view 78 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '78 more authors' ? 'Laurent Frantz, Mikkel-Holger S Sinding, Erik Ersmark, Saskia Pfrengle, Molly Cassatt-Johnstone, Ophélie Lebrasseur, Linus Girdland-Flink, Daniel M Fernandes, Morgane Ollivier, Leo Speidel, Shyam Gopalakrishnan, Michael V Westbury, Jazmin Ramos-Madrigal, Tatiana R Feuerborn, Ella Reiter, Joscha Gretzinger, Susanne C Münzel, Pooja Swali, Nicholas J Conard, Christian Carøe, James Haile, Anna Linderholm, Semyon Androsov, Ian Barnes, Chris Baumann, Norbert Benecke, Hervé Bocherens, Selina Brace, Ruth F Carden, Dorothée G Drucker, Sergey Fedorov, Mihály Gasparik, Mietje Germonpré, Semyon Grigoriev, Pam Groves, Stefan T Hertwig, Varvara V Ivanova, Luc Janssens, Richard P Jennings, Aleksei K Kasparov, Irina V Kirillova, Islam Kurmaniyazov, Yaroslav V Kuzmin, Pavel A Kosintsev, Martina Lázničková-Galetová, Charlotte Leduc, Pavel Nikolskiy, Marc Nussbaumer, Cóilı́n O’Drisceoil, Ludovic Orlando, Alan Outram, Elena Y Pavlova, Angela R Perri, Małgorzata Pilot, Vladimir V Pitulko, Valerii V Plotnikov, Albert V Protopopov, André Rehazek, Mikhail Sablin, Andaine Seguin-Orlando, Jan Storå, Christian Verjux, Victor F Zaibert, Grant Zazula, Philippe Crombé, Anders J Hansen, Eske Willerslev, Jennifer A Leonard, Anders Götherström, Ron Pinhasi, Verena J Schuenemann, Michael Hofreiter, M Thomas P Gilbert, Beth Shapiro, Greger Larson, Johannes Krause, Love Dalén, Pontus Skoglund' : '78 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">78 more authors</span> </div> <div class="periodical"> <em>Nature</em>, Jul 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>The grey wolf (Canis lupus) was the first species to give rise to a domestic population, and they remained widespread throughout the last Ice Age when many other large mammal species went extinct. Little is known, however, about the history and possible extinction of past wolf populations or when and where the wolf progenitors of the present-day dog lineage (Canis familiaris) lived1-8. Here we analysed 72 ancient wolf genomes spanning the last 100,000 years from Europe, Siberia and North America. We found that wolf populations were highly connected throughout the Late Pleistocene, with levels of differentiation an order of magnitude lower than they are today. This population connectivity allowed us to detect natural selection across the time series, including rapid fixation of mutations in the gene IFT88 40,000-30,000 years ago. We show that dogs are overall more closely related to ancient wolves from eastern Eurasia than to those from western Eurasia, suggesting a domestication process in the east. However, we also found that dogs in the Near East and Africa derive up to half of their ancestry from a distinct population related to modern southwest Eurasian wolves, reflecting either an independent domestication process or admixture from local wolves. None of the analysed ancient wolf genomes is a direct match for either of these dog ancestries, meaning that the exact progenitor populations remain to be located.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Neves2022-zt" class="col-sm-8"> <div class="title">Differential expression profiling of heat stressed tardigrades reveals major shift in the transcriptome</div> <div class="author"> Ricardo Cardoso Neves, Ask Møbjerg, Miyako Kodama, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Jazmı́n Ramos-Madrigal, M Thomas P Gilbert, Nadja Møbjerg' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Comp. Biochem. Physiol. A Mol. Integr. Physiol.</em>, May 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Tardigrades are renowned for their extreme stress tolerance, which includes the ability to endure complete desiccation, high levels of radiation and very low sub-zero temperatures. Nevertheless, tardigrades appear to be vulnerable to high temperatures and thus the potential effects of global warming. Here, we provide the first analysis of transcriptome data obtained from heat stressed specimens of the eutardigrade Ramazzottius varieornatus, with the aim of providing new insights into the molecular processes affected by high temperatures. Specifically, we compare RNA-seq datasets obtained from active, heat-exposed (35 °C) tardigrades to that of active controls kept at 5 °C. Our data reveal a surprising shift in transcription, involving 9634 differentially expressed transcripts, corresponding to &gt;35% of the transcriptome. The latter data are in striking contrast to the hitherto observed constitutive expression underlying tardigrade extreme stress tolerance and entrance into the latent state of life, known as cryptobiosis. Thus, when examining the molecular response, heat-stress appears to be more stressful for R. varieornatus than extreme conditions, such as desiccation or freezing. A gene ontology analysis reveals that the heat stress response involves a change in transcription and presumably translation, including an adjustment of metabolism, and, putatively, preparation for encystment and subsequent diapause. Among the differentially expressed transcripts we find heat-shock proteins as well as the eutardigrade specific proteins (CAHS, SAHS, MAHS, RvLEAM, and Dsup). The latter proteins thus seem to contribute to a general stress response, and may not be directly related to cryptobiosis.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Mobjerg2022-yz" class="col-sm-8"> <div class="title">Extreme freeze-tolerance in cryophilic tardigrades relies on controlled ice formation but does not involve significant change in transcription</div> <div class="author"> Ask Møbjerg, Miyako Kodama, Jazmı́n Ramos-Madrigal, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Ricardo Cardoso Neves, Aslak Jørgensen, Morten Schiøtt, M Thomas P Gilbert, Nadja Møbjerg' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">5 more authors</span> </div> <div class="periodical"> <em>Comp. Biochem. Physiol. A Mol. Integr. Physiol.</em>, Sep 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Subzero temperatures are among the most significant factors defining the distribution of organisms, yet, certain taxa have evolved to overcome this barrier. The microscopic tardigrades are among the most freeze-tolerant animals, with selected species reported to survive milli-Kelvin temperatures. Here, we estimate survival of fully hydrated eutardigrades of the species Ramazzottius varieornatus following exposures to -20 °C and -80 °C as well as -196 °C with or without initial cooling to -80 °C. The tardigrades easily survive these temperatures, yet with a significant decrease in viability following rapid cooling by direct exposure to -196 °C. Hence, post-freeze recovery of R. varieornatus seems to rely on cooling rate and thus controlled ice formation. Cryophilic organisms are renowned for having cold-active enzymes that secure appropriate reaction rates at low temperatures. Hence, extreme freeze-tolerance in R. varieornatus could potentially involve syntheses of cryoprotectants and de novo transcription. We therefore generated a reference transcriptome for this cryophilic R. varieornatus population and explored for differential gene expression patterns following cooling to -80 °C as compared to active 5 °C controls. Specifically, we tested for fast transcription potentially occurring within 25 min of cooling from room temperature to a supercooling point of ca. -20 °C, at which the tardigrades presumably freeze and enter into the ametabolic state of cryobiosis. Our analyses revealed no evidence for differential gene expression. We, therefore, conclude that extreme freeze-tolerance in R. varieornatus relies on controlled extracellular freezing with any freeze-tolerance related genes being constitutively expressed.</p> </div> </div> </div> </li> </ol> <h2 class="year">2021</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Niemann2021-wo" class="col-sm-8"> <div class="title">Extended survival of Pleistocene Siberian wolves into the early 20th century on the island of Honshū</div> <div class="author"> Jonas Niemann, Shyam Gopalakrishnan, Nobuyuki Yamaguchi, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Jazmı́n Ramos-Madrigal, Nathan Wales, M Thomas P Gilbert, Mikkel-Holger S Sinding' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>iScience</em>, Jan 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>The Japanese or Honshū wolf was one the most distinct gray wolf subspecies due to its small stature and endemicity to the islands of Honshū, Shikoku, and Kyūshū. Long revered as a guardian of farmers and travellers, it was persecuted from the 17th century following a rabies epidemic, which led to its extinction in the early 20th century. To better understand its evolutionary history, we sequenced the nuclear genome of a 19th century Honshū wolf specimen to an average depth of coverage of 3.7✕. We find Honshū wolves were closely related to a lineage of Siberian wolves that were previously believed to have gone extinct in the Late Pleistocene, thereby extending the survival of this ancient lineage until the early 20th century. We also detected significant gene flow between Japanese dogs and the Honshū wolf, corroborating previous reports on Honshū wolf dog interbreeding.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Garcia-Ortiz2021-al" class="col-sm-8"> <div class="title">The genomic landscape of Mexican Indigenous populations brings insights into the peopling of the Americas</div> <div class="author"> Humberto Garcı́a-Ortiz, Francisco Barajas-Olmos, Cecilia Contreras-Cubas, and <span class="more-authors" title="click to view 37 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '37 more authors' ? 'Miguel Ángel Cid-Soto, Emilio J Córdova, Federico Centeno-Cruz, Elvia Mendoza-Caamal, Isabel Cicerón-Arellano, Marlen Flores-Huacuja, Paulina Baca, Deborah A Bolnick, Meradeth Snow, Silvia Esperanza Flores-Martı́nez, Rocio Ortiz-Lopez, Austin W Reynolds, Antonio Blanchet, Mirna Morales-Marı́n, Rafael Velázquez-Cruz, Aleksandar David Kostic, Carlos Galaviz-Hernández, Alejandra Guadalupe Garcı́a-Zapién, José Concepción Jiménez-López, Guadalupe León-Reyes, Eva Gabriela Salas-Bautista, Blanca Patricia Lazalde-Ramos, Juan Luis Jiménez-Ruı́z, Guadalupe Salas-Martı́nez, Jazmı́n Ramos-Madrigal, Elaheh Mirzaeicheshmeh, Yolanda Saldaña-Alvarez, Marı́a Del Carmen Abrahantes-Pérez, Francisco Loeza-Becerra, Raúl Mojica-Espinosa, Federico Sánchez-Quinto, Héctor Rangel-Villalobos, Martha Sosa-Macı́as, José Sánchez-Corona, Augusto Rojas-Martinez, Angélica Martı́nez-Hernández, Lorena Orozco' : '37 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">37 more authors</span> </div> <div class="periodical"> <em>Nat. Commun.</em>, Oct 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>The genetic makeup of Indigenous populations inhabiting Mexico has been strongly influenced by geography and demographic history. Here, we perform a genome-wide analysis of 716 newly genotyped individuals from 60 of the 68 recognized ethnic groups in Mexico. We show that the genetic structure of these populations is strongly influenced by geography, and our demographic reconstructions suggest a decline in the population size of all tested populations in the last 15-30 generations. We find evidence that Aridoamerican and Mesoamerican populations diverged roughly 4-9.9 ka, around the time when sedentary farming started in Mesoamerica. Comparisons with ancient genomes indicate that the Upward Sun River 1 (USR1) individual is an outgroup to Mexican/South American Indigenous populations, whereas Anzick-1 was more closely related to Mesoamerican/South American populations than to those from Aridoamerica, showing an even more complex history of divergence than recognized so far.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Sinding2021-gf" class="col-sm-8"> <div class="title">Kouprey (Bos sauveli) genomes unveil polytomic origin of wild Asian Bos</div> <div class="author"> Mikkel-Holger S Sinding, Marta M Ciucani, Jazmı́n Ramos-Madrigal, and <span class="more-authors" title="click to view 13 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '13 more authors' ? 'Alberto Carmagnini, Jacob Agerbo Rasmussen, Shaohong Feng, Guangji Chen, Filipe G Vieira, Valeria Mattiangeli, Rajinder K Ganjoo, Greger Larson, Thomas Sicheritz-Pontén, Bent Petersen, Laurent Frantz, M Thomas P Gilbert, Daniel G Bradley' : '13 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">13 more authors</span> </div> <div class="periodical"> <em>iScience</em>, Oct 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>The evolution of the genera Bos and Bison and the nature of gene flow between wild and domestic species, is poorly understood, with genomic data of wild species being limited. We generated two genomes from the likely extinct Kouprey (Bos sauveli) and analysed them alongside other Bos and Bison genomes. We found that B. sauveli possessed genomic signatures characteristic of an independent species closely related to Bos javanicus and Bos gaurus. We found evidence for extensive incomplete lineage sorting across the three species, consistent with a polytomic diversification of the major ancestry in the group, potentially followed by secondary gene flow. Finally, we detected significant gene flow from an unsampled Asian Bos-like source into East Asian zebu cattle, demonstrating both that the full genomic diversity and evolutionary history of the Bos complex has yet to be elucidated, and that museum specimens and ancient DNA are valuable resources to do so.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Sanchez-Barreiro2021-tz" class="col-sm-8"> <div class="title">Historical population declines prompted significant genomic erosion in the northern and southern white rhinoceros (Ceratotherium simum)</div> <div class="author"> Fátima Sánchez-Barreiro, Shyam Gopalakrishnan, Jazmı́n Ramos-Madrigal, and <span class="more-authors" title="click to view 15 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '15 more authors' ? 'Michael V Westbury, Marc Manuel, Ashot Margaryan, Marta M Ciucani, Filipe G Vieira, Yannis Patramanis, Daniela C Kalthoff, Zena Timmons, Thomas Sicheritz-Pontén, Love Dalén, Oliver A Ryder, Guojie Zhang, Tomás Marquès-Bonet, Yoshan Moodley, M Thomas P Gilbert' : '15 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">15 more authors</span> </div> <div class="periodical"> <em>Mol. Ecol.</em>, Jun 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Large vertebrates are extremely sensitive to anthropogenic pressure, and their populations are declining fast. The white rhinoceros (Ceratotherium simum) is a paradigmatic case: this African megaherbivore has suffered a remarkable decline in the last 150 years due to human activities. Its subspecies, the northern (NWR) and the southern white rhinoceros (SWR), however, underwent opposite fates: the NWR vanished quickly, while the SWR recovered after the severe decline. Such demographic events are predicted to have an erosive effect at the genomic level, linked to the extirpation of diversity, and increased genetic drift and inbreeding. However, there is little empirical data available to directly reconstruct the subtleties of such processes in light of distinct demographic histories. Therefore, we generated a whole-genome, temporal data set consisting of 52 resequenced white rhinoceros genomes, representing both subspecies at two time windows: before and during/after the bottleneck. Our data reveal previously unknown population structure within both subspecies, as well as quantifiable genomic erosion. Genome-wide heterozygosity decreased significantly by 10% in the NWR and 36% in the SWR, and inbreeding coefficients rose significantly by 11% and 39%, respectively. Despite the remarkable loss of genomic diversity and recent inbreeding it suffered, the only surviving subspecies, the SWR, does not show a significant accumulation of genetic load compared to its historical counterpart. Our data provide empirical support for predictions about the genomic consequences of shrinking populations, and our findings have the potential to inform the conservation efforts of the remaining white rhinoceroses.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Feuerborn2021-wp" class="col-sm-8"> <div class="title">Modern Siberian dog ancestry was shaped by several thousand years of Eurasian-wide trade and human dispersal</div> <div class="author"> Tatiana R Feuerborn, Alberto Carmagnini, Robert J Losey, and <span class="more-authors" title="click to view 34 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '34 more authors' ? 'Tatiana Nomokonova, Arthur Askeyev, Igor Askeyev, Oleg Askeyev, Ekaterina E Antipina, Martin Appelt, Olga P Bachura, Fiona Beglane, Daniel G Bradley, Kevin G Daly, Shyam Gopalakrishnan, Kristian Murphy Gregersen, Chunxue Guo, Andrei V Gusev, Carleton Jones, Pavel A Kosintsev, Yaroslav V Kuzmin, Valeria Mattiangeli, Angela R Perri, Andrei V Plekhanov, Jazmı́n Ramos-Madrigal, Anne Lisbeth Schmidt, Dilyara Shaymuratova, Oliver Smith, Lilia V Yavorskaya, Guojie Zhang, Eske Willerslev, Morten Meldgaard, M Thomas P Gilbert, Greger Larson, Love Dalén, Anders J Hansen, Mikkel-Holger S Sinding, Laurent Frantz' : '34 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">34 more authors</span> </div> <div class="periodical"> <em>Proc. Natl. Acad. Sci. U. S. A.</em>, Sep 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Dogs have been essential to life in the Siberian Arctic for over 9,500 y, and this tight link between people and dogs continues in Siberian communities. Although Arctic Siberian groups such as the Nenets received limited gene flow from neighboring groups, archaeological evidence suggests that metallurgy and new subsistence strategies emerged in Northwest Siberia around 2,000 y ago. It is unclear if the Siberian Arctic dog population was as continuous as the people of the region or if instead admixture occurred, possibly in relation to the influx of material culture from other parts of Eurasia. To address this question, we sequenced and analyzed the genomes of 20 ancient and historical Siberian and Eurasian Steppe dogs. Our analyses indicate that while Siberian dogs were genetically homogenous between 9,500 to 7,000 y ago, later introduction of dogs from the Eurasian Steppe and Europe led to substantial admixture. This is clearly the case in the Iamal-Nenets region (Northwestern Siberia) where dogs from the Iron Age period (∼2,000 y ago) possess substantially less ancestry related to European and Steppe dogs than dogs from the medieval period (∼1,000 y ago). Combined with findings of nonlocal materials recovered from these archaeological sites, including glass beads and metal items, these results indicate that Northwest Siberian communities were connected to a larger trade network through which they acquired genetically distinctive dogs from other regions. These exchanges were part of a series of major societal changes, including the rise of large-scale reindeer pastoralism ∼800 y ago.</p> </div> </div> </div> </li> </ol> <h2 class="year">2020</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Barnett2020-ra" class="col-sm-8"> <div class="title">Genomic Adaptations and Evolutionary History of the Extinct Scimitar-Toothed Cat, Homotherium latidens</div> <div class="author"> Ross Barnett, Michael V Westbury, Marcela Sandoval-Velasco, and <span class="more-authors" title="click to view 28 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '28 more authors' ? 'Filipe Garrett Vieira, Sungwon Jeon, Grant Zazula, Michael D Martin, Simon Y W Ho, Niklas Mather, Shyam Gopalakrishnan, Jazmı́n Ramos-Madrigal, Marc Manuel, M Lisandra Zepeda-Mendoza, Agostinho Antunes, Aldo Carmona Baez, Binia De Cahsan, Greger Larson, Stephen J O’Brien, Eduardo Eizirik, Warren E Johnson, Klaus-Peter Koepfli, Andreas Wilting, Jörns Fickel, Love Dalén, Eline D Lorenzen, Tomas Marques-Bonet, Anders J Hansen, Guojie Zhang, Jong Bhak, Nobuyuki Yamaguchi, M Thomas P Gilbert' : '28 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">28 more authors</span> </div> <div class="periodical"> <em>Curr. Biol.</em>, Oct 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Homotherium was a genus of large-bodied scimitar-toothed cats, morphologically distinct from any extant felid species, that went extinct at the end of the Pleistocene [1-4]. They possessed large, saber-form serrated canine teeth, powerful forelimbs, a sloping back, and an enlarged optic bulb, all of which were key characteristics for predation on Pleistocene megafauna [5]. Previous mitochondrial DNA phylogenies suggested that it was a highly divergent sister lineage to all extant cat species [6-8]. However, mitochondrial phylogenies can be misled by hybridization [9], incomplete lineage sorting (ILS), or sex-biased dispersal patterns [10], which might be especially relevant for Homotherium since widespread mito-nuclear discrepancies have been uncovered in modern cats [10]. To examine the evolutionary history of Homotherium, we generated a ∼7x nuclear genome and a ∼38x exome from H. latidens using shotgun and target-capture sequencing approaches. Phylogenetic analyses reveal Homotherium as highly divergent (∼22.5 Ma) from living cat species, with no detectable signs of gene flow. Comparative genomic analyses found signatures of positive selection in several genes, including those involved in vision, cognitive function, and energy consumption, putatively consistent with diurnal activity, well-developed social behavior, and cursorial hunting [5]. Finally, we uncover relatively high levels of genetic diversity, suggesting that Homotherium may have been more abundant than the limited fossil record suggests [3, 4, 11-14]. Our findings complement and extend previous inferences from both the fossil record and initial molecular studies, enhancing our understanding of the evolution and ecology of this remarkable lineage.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Ramos-Madrigal2020-yn" class="col-sm-8"> <div class="title">Genomes of Pleistocene Siberian Wolves Uncover Multiple Extinct Wolf Lineages</div> <div class="author"> Jazmı́n Ramos-Madrigal, Mikkel-Holger S Sinding, Christian Carøe, and <span class="more-authors" title="click to view 24 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '24 more authors' ? 'Sarah S T Mak, Jonas Niemann, José A Samaniego Castruita, Sergey Fedorov, Alexander Kandyba, Mietje Germonpré, Hervé Bocherens, Tatiana R Feuerborn, Vladimir V Pitulko, Elena Y Pavlova, Pavel A Nikolskiy, Aleksei K Kasparov, Varvara V Ivanova, Greger Larson, Laurent A F Frantz, Eske Willerslev, Morten Meldgaard, Bent Petersen, Thomas Sicheritz-Ponten, Lutz Bachmann, Øystein Wiig, Anders J Hansen, M Thomas P Gilbert, Shyam Gopalakrishnan' : '24 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">24 more authors</span> </div> <div class="periodical"> <em>Curr. Biol.</em>, Oct 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Extant Canis lupus genetic diversity can be grouped into three phylogenetically distinct clades: Eurasian and American wolves and domestic dogs.1 Genetic studies have suggested these groups trace their origins to a wolf population that expanded during the last glacial maximum (LGM)1-3 and replaced local wolf populations.4 Moreover, ancient genomes from the Yana basin and the Taimyr peninsula provided evidence of at least one extinct wolf lineage that dwelled in Siberia during the Pleistocene.35 Previous studies have suggested that Pleistocene Siberian canids can be classified into two groups based on cranial morphology. Wolves in the first group are most similar to present-day populations, although those in the second group possess intermediate features between dogs and wolves.67 However, whether this morphological classification represents distinct genetic groups remains unknown. To investigate this question and the relationships between Pleistocene canids, present-day wolves, and dogs, we resequenced the genomes of four Pleistocene canids from Northeast Siberia dated between &gt;50 and 14 ka old, including samples from the two morphological categories. We found these specimens cluster with the two previously sequenced Pleistocene wolves, which are genetically more similar to Eurasian wolves. Our results show that, though the four specimens represent extinct wolf lineages, they do not form a monophyletic group. Instead, each Pleistocene Siberian canid branched off the lineage that gave rise to present-day wolves and dogs. Finally, our results suggest the two previously described morphological groups could represent independent lineages similarly related to present-day wolves and dogs.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Sinding2020-jw" class="col-sm-8"> <div class="title">Arctic-adapted dogs emerged at the Pleistocene-Holocene transition</div> <div class="author"> Mikkel-Holger S Sinding, Shyam Gopalakrishnan, Jazmı́n Ramos-Madrigal, and <span class="more-authors" title="click to view 32 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '32 more authors' ? 'Marc Manuel, Vladimir V Pitulko, Lukas Kuderna, Tatiana R Feuerborn, Laurent A F Frantz, Filipe G Vieira, Jonas Niemann, Jose A Samaniego Castruita, Christian Carøe, Emilie U Andersen-Ranberg, Peter D Jordan, Elena Y Pavlova, Pavel A Nikolskiy, Aleksei K Kasparov, Varvara V Ivanova, Eske Willerslev, Pontus Skoglund, Merete Fredholm, Sanne Eline Wennerberg, Mads Peter Heide-Jørgensen, Rune Dietz, Christian Sonne, Morten Meldgaard, Love Dalén, Greger Larson, Bent Petersen, Thomas Sicheritz-Pontén, Lutz Bachmann, Øystein Wiig, Tomas Marques-Bonet, Anders J Hansen, M Thomas P Gilbert' : '32 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">32 more authors</span> </div> <div class="periodical"> <em>Science</em>, Jun 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Although sled dogs are one of the most specialized groups of dogs, their origin and evolution has received much less attention than many other dog groups. We applied a genomic approach to investigate their spatiotemporal emergence by sequencing the genomes of 10 modern Greenland sled dogs, an  9500-year-old Siberian dog associated with archaeological evidence for sled technology, and an  33,000-year-old Siberian wolf. We found noteworthy genetic similarity between the ancient dog and modern sled dogs. We detected gene flow from Pleistocene Siberian wolves, but not modern American wolves, to present-day sled dogs. The results indicate that the major ancestry of modern sled dogs traces back to Siberia, where sled dog-specific haplotypes of genes that potentially relate to Arctic adaptation were established by 9500 years ago.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Moller2020-re" class="col-sm-8"> <div class="title">Near-Random Distribution of Chromosome-Derived Circular DNA in the Condensed Genome of Pigeons and the Larger, More Repeat-Rich Human Genome</div> <div class="author"> Henrik Devitt Møller, Jazmı́n Ramos-Madrigal, Iñigo Prada-Luengo, and <span class="more-authors" title="click to view 2 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '2 more authors' ? 'M Thomas P Gilbert, Birgitte Regenberg' : '2 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">2 more authors</span> </div> <div class="periodical"> <em>Genome Biol. Evol.</em>, Jan 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Extrachromosomal circular DNA (eccDNA) elements of chromosomal origin are known to be common in a number of eukaryotic species. However, it remains to be addressed whether genomic features such as genome size, the load of repetitive elements within a genome, and/or animal physiology affect the number of eccDNAs. Here, we investigate the distribution and numbers of eccDNAs in a condensed and less repeat-rich genome compared with the human genome, using Columba livia domestica (domestic rock pigeon) as a model organism. By sequencing eccDNA in blood and breast muscle from three pigeon breeds at various ages and with different flight behavior, we characterize 30,000 unique eccDNAs. We identify genomic regions that are likely hotspots for DNA circularization in breast muscle, including genes involved in muscle development. We find that although eccDNA counts do not correlate with the biological age in pigeons, the number of unique eccDNAs in a nonflying breed (king pigeons) is significantly higher (9-fold) than homing pigeons. Furthermore, a comparison between eccDNA from skeletal muscle in pigeons and humans reveals ∼9-10 times more unique eccDNAs per human nucleus. The fraction of eccDNA sequences, derived from repetitive elements, exist in proportions to genome content, that is, human 72.4% (expected 52.5%) and pigeon 8.7% (expected 5.5%). Overall, our results support that eccDNAs are common in pigeons, that the amount of unique eccDNA types per nucleus can differ between species as well as subspecies, and suggest that eccDNAs from repeats are found in proportions relative to the content of repetitive elements in a genome.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Nagele2020-ff" class="col-sm-8"> <div class="title">Genomic insights into the early peopling of the Caribbean</div> <div class="author"> Kathrin Nägele, Cosimo Posth, Miren Iraeta Orbegozo, and <span class="more-authors" title="click to view 26 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '26 more authors' ? 'Yadira Armas, Silvia Teresita Hernández Godoy, Ulises M González Herrera, Maria A Nieves-Colón, Marcela Sandoval-Velasco, Dorothea Mylopotamitaki, Rita Radzeviciute, Jason Laffoon, William J Pestle, Jazmin Ramos-Madrigal, Thiseas C Lamnidis, William C Schaffer, Robert S Carr, Jane S Day, Carlos Arredondo Antúnez, Armando Rangel Rivero, Antonio J Martı́nez-Fuentes, Edwin Crespo-Torres, Ivan Roksandic, Anne C Stone, Carles Lalueza-Fox, Menno Hoogland, Mirjana Roksandic, Corinne L Hofman, Johannes Krause, Hannes Schroeder' : '26 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">26 more authors</span> </div> <div class="periodical"> <em>Science</em>, Jul 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>The Caribbean was one of the last regions of the Americas to be settled by humans, but where they came from and how and when they reached the islands remain unclear. We generated genome-wide data for 93 ancient Caribbean islanders dating between 3200 and 400 calibrated years before the present and found evidence of at least three separate dispersals into the region, including two early dispersals into the Western Caribbean, one of which seems connected to radiation events in North America. This was followed by a later expansion from South America. We also detected genetic differences between the early settlers and the newcomers from South America, with almost no evidence of admixture. Our results add to our understanding of the initial peopling of the Caribbean and the movements of Archaic Age peoples in the Americas.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Welker2020-dq" class="col-sm-8"> <div class="title">The dental proteome of Homo antecessor</div> <div class="author"> Frido Welker, Jazmı́n Ramos-Madrigal, Petra Gutenbrunner, and <span class="more-authors" title="click to view 23 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '23 more authors' ? 'Meaghan Mackie, Shivani Tiwary, Rosa Rakownikow Jersie-Christensen, Cristina Chiva, Marc R Dickinson, Martin Kuhlwilm, Marc Manuel, Pere Gelabert, Marı́a Martinón-Torres, Ann Margvelashvili, Juan Luis Arsuaga, Eudald Carbonell, Tomas Marques-Bonet, Kirsty Penkman, Eduard Sabidó, Jürgen Cox, Jesper V Olsen, David Lordkipanidze, Fernando Racimo, Carles Lalueza-Fox, José Marı́a Castro, Eske Willerslev, Enrico Cappellini' : '23 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">23 more authors</span> </div> <div class="periodical"> <em>Nature</em>, Apr 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>The phylogenetic relationships between hominins of the Early Pleistocene epoch in Eurasia, such as Homo antecessor, and hominins that appear later in the fossil record during the Middle Pleistocene epoch, such as Homo sapiens, are highly debated1-5. For the oldest remains, the molecular study of these relationships is hindered by the degradation of ancient DNA. However, recent research has demonstrated that the analysis of ancient proteins can address this challenge6-8. Here we present the dental enamel proteomes of H. antecessor from Atapuerca (Spain)9,10 and Homo erectus from Dmanisi (Georgia)1, two key fossil assemblages that have a central role in models of Pleistocene hominin morphology, dispersal and divergence. We provide evidence that H. antecessor is a close sister lineage to subsequent Middle and Late Pleistocene hominins, including modern humans, Neanderthals and Denisovans. This placement implies that the modern-like face of H. antecessor-that is, similar to that of modern humans-may have a considerably deep ancestry in the genus Homo, and that the cranial morphology of Neanderthals represents a derived form. By recovering AMELY-specific peptide sequences, we also conclude that the H. antecessor molar fragment from Atapuerca that we analysed belonged to a male individual. Finally, these H. antecessor and H. erectus fossils preserve evidence of enamel proteome phosphorylation and proteolytic digestion that occurred in vivo during tooth formation. Our results provide important insights into the evolutionary relationships between H. antecessor and other hominin groups, and pave the way for future studies using enamel proteomes to investigate hominin biology across the existence of the genus Homo.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Kistler2020-vo" class="col-sm-8"> <div class="title">Ancient Plant Genomics in Archaeology, Herbaria, and the Environment</div> <div class="author"> Logan Kistler, Vanessa C Bieker, Michael D Martin, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Mikkel Winther Pedersen, Jazmı́n Ramos Madrigal, Nathan Wales' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>Annu. Rev. Plant Biol.</em>, Apr 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>The ancient DNA revolution of the past 35 years has driven an explosion in the breadth, nuance, and diversity of questions that are approachable using ancient biomolecules, and plant research has been a constant, indispensable facet of these developments. Using archaeological, paleontological, and herbarium plant tissues, researchers have probed plant domestication and dispersal, plant evolution and ecology, paleoenvironmental composition and dynamics, and other topics across related disciplines. Here, we review the development of the ancient DNA discipline and the role of plant research in its progress and refinement. We summarize our understanding of long-term plant DNA preservation and the characteristics of degraded DNA. In addition, we discuss challenges in ancient DNA recovery and analysis and the laboratory and bioinformatic strategies used to mitigate them. Finally, we review recent applications of ancient plant genomic research.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Bouby2020-cv" class="col-sm-8"> <div class="title">Tracking the history of grapevine cultivation in Georgia by combining geometric morphometrics and ancient DNA</div> <div class="author"> Laurent Bouby, Nathan Wales, Mindia Jalabadze, and <span class="more-authors" title="click to view 12 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '12 more authors' ? 'Nana Rusishvili, Vincent Bonhomme, Jazmı́n Ramos-Madrigal, Allowen Evin, Sarah Ivorra, Thierry Lacombe, Clémence Pagnoux, Elisabetta Boaretto, M Thomas P Gilbert, Roberto Bacilieri, David Lordkipanidze, David Maghradze' : '12 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">12 more authors</span> </div> <div class="periodical"> <em>Veg. Hist. Archaeobot.</em>, Oct 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>The Near East and the Caucasus are commonly regarded as the original domestication centres of Vitis vinifera (grapevine), and the region continues to be home to a high diversity of wild and cultivated grapevines, particularly within Georgia. The earliest chemical evidence for wine making was recorded in Georgian Neolithic sites (6000–5800 bc) and grape pips, possibly of the domesticated morphotype, have been reported from several sites of about the same period. We performed geometric morphometric and palaeogenomic investigations of grape pip samples in order to identify the appearance of domesticated grapevine and explore the changes in cultivated diversity in relation to modern varieties. We systematically investigated charred and uncharred grape pip samples from Georgian archaeological sites. Their chronology was thoroughly assessed by direct radiocarbon dating. More than 500 grape pips from 14 sites from the Middle Bronze Age to modern times were selected for geometric morphometric studies. The shapes of the ancient pips were compared to hundreds of modern wild individuals and cultivated varieties. Degraded DNA was isolated from three pips from two sites, converted to Illumina libraries, sequenced at approximately 10,000 single nucleotide polymorphism (SNP) sites, and compared to a large public database of grapevine diversity. The most ancient pip dates from the Middle Bronze Age (1900–1500 cal bc) and the domesticated morphotype is identified from ca. 1000 bc onwards. A great diversity of domesticated shapes was regularly seen in the samples. Most are close to modern cultivars from the Caucasian, southwest Asian and Balkan areas, which suggests that the modern local vine diversity is deeply rooted in early viticulture. DNA was successfully recovered from historic pips and genome-wide analyses found close parental relationships to modern Georgian cultivars.</p> </div> </div> </div> </li> </ol> <h2 class="year">2019</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Ramos-Madrigal2019-lf" class="col-sm-8"> <div class="title">Palaeogenomic insights into the origins of French grapevine diversity</div> <div class="author"> Jazmı́n Ramos-Madrigal, Anne Kathrine Wiborg Runge, Laurent Bouby, and <span class="more-authors" title="click to view 14 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '14 more authors' ? 'Thierry Lacombe, José Alfredo Samaniego Castruita, Anne-Françoise Adam-Blondon, Isabel Figueiral, Charlotte Hallavant, José M Martı́nez-Zapater, Caroline Schaal, Reinhard Töpfer, Bent Petersen, Thomas Sicheritz-Pontén, Patrice This, Roberto Bacilieri, M Thomas P Gilbert, Nathan Wales' : '14 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">14 more authors</span> </div> <div class="periodical"> <em>Nat Plants</em>, Jun 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>The Eurasian grapevine (Vitis vinifera) has long been important for wine production as well as being a food source. Despite being clonally propagated, modern cultivars exhibit great morphological and genetic diversity, with thousands of varieties described in historic and contemporaneous records. Through historical accounts, some varieties can be traced to the Middle Ages, but the genetic relationships between ancient and modern vines remain unknown. We present target-enriched genome-wide sequencing data from 28 archaeological grape seeds dating to the Iron Age, Roman era and medieval period. When compared with domesticated and wild accessions, we found that the archaeological samples were closely related to western European cultivars used for winemaking today. We identified seeds with identical genetic signatures present at different Roman sites, as well as seeds sharing parent-offspring relationships with varieties grown today. Furthermore, we discovered that one seed dated to  1100 CE was a genetic match to ’Savagnin Blanc’, providing evidence for 900 years of uninterrupted vegetative propagation.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Cappellini2019-xy" class="col-sm-8"> <div class="title">Early Pleistocene enamel proteome from Dmanisi resolves Stephanorhinus phylogeny</div> <div class="author"> Enrico Cappellini, Frido Welker, Luca Pandolfi, and <span class="more-authors" title="click to view 40 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '40 more authors' ? 'Jazmı́n Ramos-Madrigal, Diana Samodova, Patrick L Rüther, Anna K Fotakis, David Lyon, J Vı́ctor Moreno-Mayar, Maia Bukhsianidze, Rosa Rakownikow Jersie-Christensen, Meaghan Mackie, Aurélien Ginolhac, Reid Ferring, Martha Tappen, Eleftheria Palkopoulou, Marc R Dickinson, Thomas W Stafford, Yvonne L Chan, Anders Götherström, Senthilvel K S S Nathan, Peter D Heintzman, Joshua D Kapp, Irina Kirillova, Yoshan Moodley, Jordi Agusti, Ralf-Dietrich Kahlke, Gocha Kiladze, Bienvenido Martı́nez-Navarro, Shanlin Liu, Marcela Sandoval Velasco, Mikkel-Holger S Sinding, Christian D Kelstrup, Morten E Allentoft, Ludovic Orlando, Kirsty Penkman, Beth Shapiro, Lorenzo Rook, Love Dalén, M Thomas P Gilbert, Jesper V Olsen, David Lordkipanidze, Eske Willerslev' : '40 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">40 more authors</span> </div> <div class="periodical"> <em>Nature</em>, Oct 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>The sequencing of ancient DNA has enabled the reconstruction of speciation, migration and admixture events for extinct taxa1. However, the irreversible post-mortem degradation2 of ancient DNA has so far limited its recovery-outside permafrost areas-to specimens that are not older than approximately 0.5 million years (Myr)3. By contrast, tandem mass spectrometry has enabled the sequencing of approximately 1.5-Myr-old collagen type I4, and suggested the presence of protein residues in fossils of the Cretaceous period5-although with limited phylogenetic use6. In the absence of molecular evidence, the speciation of several extinct species of the Early and Middle Pleistocene epoch remains contentious. Here we address the phylogenetic relationships of the Eurasian Rhinocerotidae of the Pleistocene epoch7-9, using the proteome of dental enamel from a Stephanorhinus tooth that is approximately 1.77-Myr old, recovered from the archaeological site of Dmanisi (South Caucasus, Georgia)10. Molecular phylogenetic analyses place this Stephanorhinus as a sister group to the clade formed by the woolly rhinoceros (Coelodonta antiquitatis) and Merck’s rhinoceros (Stephanorhinus kirchbergensis). We show that Coelodonta evolved from an early Stephanorhinus lineage, and that this latter genus includes at least two distinct evolutionary lines. The genus Stephanorhinus is therefore currently paraphyletic, and its systematic revision is needed. We demonstrate that sequencing the proteome of Early Pleistocene dental enamel overcomes the limitations of phylogenetic inference based on ancient collagen or DNA. Our approach also provides additional information about the sex and taxonomic assignment of other specimens from Dmanisi. Our findings reveal that proteomic investigation of ancient dental enamel-which is the hardest tissue in vertebrates11, and is highly abundant in the fossil record-can push the reconstruction of molecular evolution further back into the Early Pleistocene epoch, beyond the currently known limits of ancient DNA preservation.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Schablitsky2019-vm" class="col-sm-8"> <div class="title">Ancient DNA analysis of a nineteenth century tobacco pipe from a Maryland slave quarter</div> <div class="author"> Julie M Schablitsky, Kelsey E Witt, Jazmı́n Ramos Madrigal, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Martin R Ellegaard, Ripan S Malhi, Hannes Schroeder' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">3 more authors</span> </div> <div class="periodical"> <em>J. Archaeol. Sci.</em>, May 2019 </div> <div class="periodical"> </div> <div class="links"> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Ziesemer2019-ek" class="col-sm-8"> <div class="title">The efficacy of whole human genome capture on ancient dental calculus and dentin</div> <div class="author"> Kirsten A Ziesemer, Jazmı́n Ramos-Madrigal, Allison E Mann, and <span class="more-authors" title="click to view 14 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '14 more authors' ? 'Bernd W Brandt, Krithivasan Sankaranarayanan, Andrew T Ozga, Menno Hoogland, Courtney A Hofman, Domingo C Salazar-Garcı́a, Bruno Frohlich, George R Milner, Anne C Stone, Mark Aldenderfer, Cecil M Lewis, Corinne L Hofman, Christina Warinner, Hannes Schroeder' : '14 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">14 more authors</span> </div> <div class="periodical"> <em>Am. J. Phys. Anthropol.</em>, Mar 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>OBJECTIVES: Dental calculus is among the richest known sources of ancient DNA in the archaeological record. Although most DNA within calculus is microbial, it has been shown to contain sufficient human DNA for the targeted retrieval of whole mitochondrial genomes. Here, we explore whether calculus is also a viable substrate for whole human genome recovery using targeted enrichment techniques. MATERIALS AND METHODS: Total DNA extracted from 24 paired archaeological human dentin and calculus samples was subjected to whole human genome enrichment using in-solution hybridization capture and high-throughput sequencing. RESULTS: Total DNA from calculus exceeded that of dentin in all cases, and although the proportion of human DNA was generally lower in calculus, the absolute human DNA content of calculus and dentin was not significantly different. Whole genome enrichment resulted in up to four-fold enrichment of the human endogenous DNA content for both dentin and dental calculus libraries, albeit with some loss in complexity. Recovering more on-target reads for the same sequencing effort generally improved the quality of downstream analyses, such as sex and ancestry estimation. For nonhuman DNA, comparison of phylum-level microbial community structure revealed few differences between precapture and postcapture libraries, indicating that off-target sequences in human genome-enriched calculus libraries may still be useful for oral microbiome reconstruction. DISCUSSION: While ancient human dental calculus does contain endogenous human DNA sequences, their relative proportion is low when compared with other skeletal tissues. Whole genome enrichment can help increase the proportion of recovered human reads, but in this instance enrichment efficiency was relatively low when compared with other forms of capture. We conclude that further optimization is necessary before the method can be routinely applied to archaeological samples.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Welker2019-xb" class="col-sm-8"> <div class="title">Enamel proteome shows that Gigantopithecus was an early diverging pongine</div> <div class="author"> Frido Welker, Jazmı́n Ramos-Madrigal, Martin Kuhlwilm, and <span class="more-authors" title="click to view 15 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '15 more authors' ? 'Wei Liao, Petra Gutenbrunner, Marc Manuel, Diana Samodova, Meaghan Mackie, Morten E Allentoft, Anne-Marie Bacon, Matthew J Collins, Jürgen Cox, Carles Lalueza-Fox, Jesper V Olsen, Fabrice Demeter, Wei Wang, Tomas Marques-Bonet, Enrico Cappellini' : '15 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">15 more authors</span> </div> <div class="periodical"> <em>Nature</em>, Dec 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Gigantopithecus blacki was a giant hominid that inhabited densely forested environments of Southeast Asia during the Pleistocene epoch1. Its evolutionary relationships to other great ape species, and the divergence of these species during the Middle and Late Miocene epoch (16-5.3 million years ago), remain unclear2,3. Hypotheses regarding the relationships between Gigantopithecus and extinct and extant hominids are wide ranging but difficult to substantiate because of its highly derived dentognathic morphology, the absence of cranial and post-cranial remains1,3-6, and the lack of independent molecular validation. We retrieved dental enamel proteome sequences from a 1.9-million-year-old G. blacki molar found in Chuifeng Cave, China7,8. The thermal age of these protein sequences is approximately five times greater than that of any previously published mammalian proteome or genome. We demonstrate that Gigantopithecus is a sister clade to orangutans (genus Pongo) with a common ancestor about 12-10 million years ago, implying that the divergence of Gigantopithecus from Pongo forms part of the Miocene radiation of great apes. In addition, we hypothesize that the expression of alpha-2-HS-glycoprotein, which has not been previously observed in enamel proteomes, had a role in the biomineralization of the thick enamel crowns that characterize the large molars in Gigantopithecus9,10. The survival of an Early Pleistocene dental enamel proteome in the subtropics further expands the scope of palaeoproteomic analysis into geographical areas and time periods previously considered incompatible with the preservation of substantial amounts of genetic information.</p> </div> </div> </div> </li> </ol> <h2 class="year">2018</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Gopalakrishnan2018-sr" class="col-sm-8"> <div class="title">Interspecific Gene Flow Shaped the Evolution of the Genus Canis</div> <div class="author"> Shyam Gopalakrishnan, Mikkel-Holger S Sinding, Jazmı́n Ramos-Madrigal, and <span class="more-authors" title="click to view 25 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '25 more authors' ? 'Jonas Niemann, Jose A Samaniego Castruita, Filipe G Vieira, Christian Carøe, Marc de Manuel Montero, Lukas Kuderna, Aitor Serres, Vı́ctor Manuel González-Basallote, Yan-Hu Liu, Guo-Dong Wang, Tomas Marques-Bonet, Siavash Mirarab, Carlos Fernandes, Philippe Gaubert, Klaus-Peter Koepfli, Jane Budd, Eli Knispel Rueness, Claudio Sillero, Mads Peter Heide-Jørgensen, Bent Petersen, Thomas Sicheritz-Ponten, Lutz Bachmann, Øystein Wiig, Anders J Hansen, M Thomas P Gilbert' : '25 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">25 more authors</span> </div> <div class="periodical"> <em>Curr. Biol.</em>, Nov 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>The evolutionary history of the wolf-like canids of the genus Canis has been heavily debated, especially regarding the number of distinct species and their relationships at the population and species level [1-6]. We assembled a dataset of 48 resequenced genomes spanning all members of the genus Canis except the black-backed and side-striped jackals, encompassing the global diversity of seven extant canid lineages. This includes eight new genomes, including the first resequenced Ethiopian wolf (Canis simensis), one dhole (Cuon alpinus), two East African hunting dogs (Lycaon pictus), two Eurasian golden jackals (Canis aureus), and two Middle Eastern gray wolves (Canis lupus). The relationships between the Ethiopian wolf, African golden wolf, and golden jackal were resolved. We highlight the role of interspecific hybridization in the evolution of this charismatic group. Specifically, we find gene flow between the ancestors of the dhole and African hunting dog and admixture between the gray wolf, coyote (Canis latrans), golden jackal, and African golden wolf. Additionally, we report gene flow from gray and Ethiopian wolves to the African golden wolf, suggesting that the African golden wolf originated through hybridization between these species. Finally, we hypothesize that coyotes and gray wolves carry genetic material derived from a “ghost” basal canid lineage.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Kistler2018-ny" class="col-sm-8"> <div class="title">Multiproxy evidence highlights a complex evolutionary legacy of maize in South America</div> <div class="author"> Logan Kistler, S Yoshi Maezumi, Jonas Souza, and <span class="more-authors" title="click to view 14 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '14 more authors' ? 'Natalia A S Przelomska, Flaviane Malaquias Costa, Oliver Smith, Hope Loiselle, Jazmı́n Ramos-Madrigal, Nathan Wales, Eduardo Rivail Ribeiro, Ryan R Morrison, Claudia Grimaldo, Andre P Prous, Bernardo Arriaza, M Thomas P Gilbert, Fabio Oliveira Freitas, Robin G Allaby' : '14 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">14 more authors</span> </div> <div class="periodical"> <em>Science</em>, Dec 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Domesticated maize evolved from wild teosinte under human influences in Mexico beginning around 9000 years before the present (yr B.P.), traversed Central America by  7500 yr B.P., and spread into South America by  6500 yr B.P. Landrace and archaeological maize genomes from South America suggest that the ancestral population to South American maize was brought out of the domestication center in Mexico and became isolated from the wild teosinte gene pool before traits of domesticated maize were fixed. Deeply structured lineages then evolved within South America out of this partially domesticated progenitor population. Genomic, linguistic, archaeological, and paleoecological data suggest that the southwestern Amazon was a secondary improvement center for partially domesticated maize. Multiple waves of human-mediated dispersal are responsible for the diversity and biogeography of modern South American maize.</p> </div> </div> </div> </li> </ol> <h2 class="year">2017</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Mohd_Salleh2017-ve" class="col-sm-8"> <div class="title">An expanded mammal mitogenome dataset from Southeast Asia</div> <div class="author"> Faezah Mohd Salleh, Jazmı́n Ramos-Madrigal, Fernando Peñaloza, and <span class="more-authors" title="click to view 13 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '13 more authors' ? 'Shanlin Liu, S Sinding Mikkel-Holger, P Patel Riddhi, Renata Martins, Dorina Lenz, Jörns Fickel, Christian Roos, Mohd Shahir Shamsir, Mohammad Shahfiz Azman, K Lim Burton, J Rossiter Stephen, Andreas Wilting, M Thomas P Gilbert' : '13 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">13 more authors</span> </div> <div class="periodical"> <em>Gigascience</em>, Aug 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>Southeast (SE) Asia is 1 of the most biodiverse regions in the world, and it holds approximately 20% of all mammal species. Despite this, the majority of SE Asia’s genetic diversity is still poorly characterized. The growing interest in using environmental DNA to assess and monitor SE Asian species, in particular threatened mammals-has created the urgent need to expand the available reference database of mitochondrial barcode and complete mitogenome sequences. We have partially addressed this need by generating 72 new mitogenome sequences reconstructed from DNA isolated from a range of historical and modern tissue samples. Approximately 55 gigabases of raw sequence were generated. From this data, we assembled 72 complete mitogenome sequences, with an average depth of coverage of \times102.9 and \times55.2 for modern samples and historical samples, respectively. This dataset represents 52 species, of which 30 species had no previous mitogenome data available. The mitogenomes were geotagged to their sampling location, where known, to display a detailed geographical distribution of the species. Our new database of 52 taxa will strongly enhance the utility of environmental DNA approaches for monitoring mammals in SE Asia as it greatly increases the likelihoods that identification of metabarcoding sequencing reads can be assigned to reference sequences. This magnifies the confidence in species detections and thus allows more robust surveys and monitoring programmes of SE Asia’s threatened mammal biodiversity. The extensive collections of historical samples from SE Asia in western and SE Asian museums should serve as additional valuable material to further enrich this reference database.</p> </div> </div> </div> </li></ol> <h2 class="year">2016</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Ramos-Madrigal2016-qy" class="col-sm-8"> <div class="title">Genome Sequence of a 5,310-Year-Old Maize Cob Provides Insights into the Early Stages of Maize Domestication</div> <div class="author"> Jazmı́n Ramos-Madrigal, Bruce D Smith, J Vı́ctor Moreno-Mayar, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Shyam Gopalakrishnan, Jeffrey Ross-Ibarra, M Thomas P Gilbert, Nathan Wales' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">4 more authors</span> </div> <div class="periodical"> <em>Curr. Biol.</em>, Dec 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>The complex evolutionary history of maize (Zea mays L. ssp. mays) has been clarified with genomic-level data from modern landraces and wild teosinte grasses [1, 2], augmenting archaeological findings that suggest domestication occurred between 10,000 and 6,250 years ago in southern Mexico [3, 4]. Maize rapidly evolved under human selection, leading to conspicuous phenotypic transformations, as well as adaptations to varied environments [5]. Still, many questions about the domestication process remain unanswered because modern specimens do not represent the full range of past diversity due to abandonment of unproductive lineages, genetic drift, on-going natural selection, and recent breeding activity. To more fully understand the history and spread of maize, we characterized the draft genome of a 5,310-year-old archaeological cob excavated in the Tehuacan Valley of Mexico. We compare this ancient sample against a reference panel of modern landraces and teosinte grasses using D statistics, model-based clustering algorithms, and multidimensional scaling analyses, demonstrating the specimen derives from the same source population that gave rise to modern maize. We find that 5,310 years ago, maize in the Tehuacan Valley was on the whole genetically closer to modern maize than to its wild counterpart. However, many genes associated with key domestication traits existed in the ancestral state, sharply contrasting with the ubiquity of derived alleles in living landraces. These findings suggest much of the evolution during domestication may have been gradual and encourage further paleogenomic research to address provocative questions about the world’s most produced cereal.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Wales2016-dp" class="col-sm-8"> <div class="title">The limits and potential of paleogenomic techniques for reconstructing grapevine domestication</div> <div class="author"> Nathan Wales, Jazmı́n Ramos Madrigal, Enrico Cappellini, and <span class="more-authors" title="click to view 24 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '24 more authors' ? 'Aldo Carmona Baez, José Alfredo Samaniego Castruita, J Alberto Romero-Navarro, Christian Carøe, Marı́a C Ávila-Arcos, Fernando Peñaloza, J Vı́ctor Moreno-Mayar, Boris Gasparyan, Diana Zardaryan, Tamara Bagoyan, Alexia Smith, Ron Pinhasi, Giovanna Bosi, Girolamo Fiorentino, Anna Maria Grasso, Alessandra Celant, Guy Bar-Oz, Yotam Tepper, Allan Hall, Simone Scalabrin, Mara Miculan, Michele Morgante, Gabriele Di Gaspero, M Thomas P Gilbert' : '24 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">24 more authors</span> </div> <div class="periodical"> <em>J. Archaeol. Sci.</em>, Aug 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>In ancient DNA (aDNA) research, evolutionary and archaeological questions are often investigated using the genomic sequences of organelles: mitochondrial and chloroplast DNA. Organellar genomes are found in multiple copies per living cell, increasing their chance of recovery from archaeological samples, and are inherited from one parent without genetic recombination, simplifying analyses. While mitochondrial genomes have played a key role in many mammalian aDNA projects, including research focused on prehistoric humans and extinct hominins, it is unclear how useful plant chloroplast genomes (plastomes) may be at elucidating questions related to plant evolution, crop domestication, and the prehistoric movement of botanical products through trade and migration. Such analyses are particularly challenging for plant species whose genomes have highly repetitive sequences and that undergo frequent genomic reorganization, notably species with high retrotransposon activity. To address this question, we explored the research potential of the grape (Vitis vinifera L.) plastome using targeted-enrichment methods and high-throughput DNA sequencing on a collection of archaeological grape pip and vine specimens from sites across Eurasia dating ca. 4000 BCE–1500 CE. We demonstrate that due to unprecedented numbers of sequence insertions into the nuclear and mitochondrial genomes, the grape plastome provides limited intraspecific phylogenetic resolution. Nonetheless, we were able to assign archaeological specimens in the Italian peninsula, Sardinia, UK, and Armenia from pre-Roman to medieval times as belonging to all three major chlorotypes A, C, and D found in modern varieties of Western Europe. Analysis of nuclear genomic DNA from these samples reveals a much greater potential for understanding ancient viticulture, including domestication events, genetic introgression from local wild populations, and the origins and histories of varietal lineages.</p> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Da_Fonseca2016-xi" class="col-sm-8"> <div class="title">Next-generation biology: Sequencing and data analysis approaches for non-model organisms</div> <div class="author"> Rute R Fonseca, Anders Albrechtsen, Gonçalo Espregueira Themudo, and <span class="more-authors" title="click to view 7 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '7 more authors' ? 'Jazmı́n Ramos-Madrigal, Jonas Andreas Sibbesen, Lasse Maretty, M Lisandra Zepeda-Mendoza, Paula F Campos, Rasmus Heller, Ricardo J Pereira' : '7 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">7 more authors</span> </div> <div class="periodical"> <em>Mar. Genomics</em>, Dec 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>As sequencing technologies become more affordable, it is now realistic to propose studying the evolutionary history of virtually any organism on a genomic scale. However, when dealing with non-model organisms it is not always easy to choose the best approach given a specific biological question, a limited budget, and challenging sample material. Furthermore, although recent advances in technology offer unprecedented opportunities for research in non-model organisms, they also demand unprecedented awareness from the researcher regarding the assumptions and limitations of each method. In this review we present an overview of the current sequencing technologies and the methods used in typical high-throughput data analysis pipelines. Subsequently, we contextualize high-throughput DNA sequencing technologies within their applications in non-model organism biology. We include tips regarding managing unconventional sample material, comparative and population genetic approaches that do not require fully assembled genomes, and advice on how to deal with low depth sequencing data.</p> </div> </div> </div> </li> </ol> <h2 class="year">2015</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"></div> <div id="Wales2015-jv" class="col-sm-8"> <div class="title">New insights on single-stranded versus double-stranded DNA library preparation for ancient DNA</div> <div class="author"> Nathan Wales, Christian Carøe, Marcela Sandoval-Velasco, and <span class="more-authors" title="click to view 6 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '6 more authors' ? 'Cristina Gamba, Ross Barnett, José Alfredo Samaniego, Jazmı́n Ramos Madrigal, Ludovic Orlando, M Thomas P Gilbert' : '6 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">6 more authors</span> </div> <div class="periodical"> <em>Biotechniques</em>, Dec 2015 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> </div> <div class="badges"> <span class="altmetric-embed" data-altmetric-id="" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right"></span> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 6px;"></span> </div> <div class="abstract hidden"> <p>An innovative single-stranded DNA (ssDNA) library preparation method has sparked great interest among ancient DNA (aDNA) researchers, especially after reports of endogenous DNA content increases &gt;20-fold in some samples. To investigate the behavior of this method, we generated ssDNA and conventional double-stranded DNA (dsDNA) libraries from 23 ancient and historic plant and animal specimens. We found ssDNA library preparation substantially increased endogenous content when dsDNA libraries contained &lt;3% endogenous DNA, but this enrichment is less pronounced when dsDNA preparations successfully recover short endogenous DNA fragments (mean size &lt; 70 bp). Our findings can help researchers determine when to utilize the time- and resource-intensive ssDNA library preparation method.</p> </div> </div> </div> </li></ol> </div> </article> </div> </div> <footer class="fixed-bottom"> <div class="container mt-0"> © Copyright 2023 Jazmín Ramos Madrigal. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Last updated: March 16, 2023. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js"></script> <script defer src="/assets/js/common.js"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>